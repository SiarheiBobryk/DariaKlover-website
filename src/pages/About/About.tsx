import * as React from 'react';
import { Helmet } from 'react-helmet';

import Typography from '@mui/material/Typography';
import Link from '@mui/material/Link';

// TODO: It would be great to add a pic here
function About() {
  return (
    <>
      <Helmet>
        <title>–û–±–æ –º–Ω–µ</title>
      </Helmet>
      <Typography variant="h4" component="h1" gutterBottom>
        –û–±–æ –º–Ω–µ
      </Typography>
      <Typography gutterBottom>–ü—Ä–∏–≤–µ—Ç! –Ø - –î–∞—Ä—å—è –ö–ª–æ–≤–µ—Ä.</Typography>
      <Typography gutterBottom>
        –ö–ª–æ–≤–µ—Ä - –º–æ–π –ø—Å–µ–≤–¥–æ–Ω–∏–º. –û—Ç –∞–Ω–≥–ª–∏–π—Å–∫–æ–≥–æ <Typography component="i">Ààkl…ô äv…ôr</Typography> - –∫–ª–µ–≤–µ—Ä üçÄ
      </Typography>
      <Typography gutterBottom>
        –ú–Ω–µ –æ—á–µ–Ω—å –Ω—Ä–∞–≤–∏—Ç—Å—è –∏–¥–µ—è –ø–µ—Ä–µ–¥–∞—á–∏ —Å–º—ã—Å–ª–æ–≤ —á–µ—Ä–µ–∑ –æ–±—Ä–∞–∑—ã. –í —ç—Ç–æ–º –µ—Å—Ç—å —á—Ç–æ-—Ç–æ —É–≤–ª–µ–∫–∞—Ç–µ–ª—å–Ω–æ–µ, –∏–Ω—Ç–µ—Ä–µ—Å–Ω–æ–µ –∏
        —Ç–∞–∏–Ω—Å—Ç–≤–µ–Ω–Ω–æ–µ. –£ –∫–ª–µ–≤–µ—Ä–∞ –µ—Å—Ç—å –º–Ω–æ–≥–æ –∑–Ω–∞—á–µ–Ω–∏–π. –î–ª—è –º–µ–Ω—è - —ç—Ç–æ –æ–ª–∏—Ü–µ—Ç–≤–æ—Ä–µ–Ω–∏–µ –±–∞–ª–∞–Ω—Å–∞ –∏ –≥–∞—Ä–º–æ–Ω–∏–∏. –ü–æ-–º–æ–µ–º—É, –∑–Ω–∞—á–µ–Ω–∏–µ
        —ç—Ç–æ–≥–æ —Å–∏–º–≤–æ–ª–∞ –æ—á–µ–Ω—å —Ö–æ—Ä–æ—à–æ –æ—Ç—Ä–∞–∂–∞–µ—Ç –º–µ–Ω—è –∏ —Ç–æ, –∫ –µ–º—É —è —Å—Ç—Ä–µ–º–ª—é—Å—å, –≤–æ —á—Ç–æ –≤–µ—Ä—é, —á–µ–º –∑–∞–Ω–∏–º–∞–ª–∞—Å—å –∏ –ø—Ä–æ–¥–æ–ª–∂–∞—é
        –∑–∞–Ω–∏–º–∞—Ç—å—Å—è.
      </Typography>
      <Typography gutterBottom>
        –í –ø—Ä–æ—à–ª–æ–º —è –≤—Ä–∞—á —Ç–µ—Ä–∞–ø–µ–≤—Ç, –≤—Ä–∞—á –æ–±—â–µ–π –ø—Ä–∞–∫—Ç–∏–∫–∏, –º–∞—Å—Å–∞–∂–∏—Å—Ç –∏ –∫–æ—Å–º–µ—Ç–∏–∫. –Ø –∑–Ω–∞—é –Ω–µ –ø–æ–Ω–∞—Å–ª—ã—à–∫–µ, —á—Ç–æ –¥–ª—è —Å–æ—Ö—Ä–∞–Ω–µ–Ω–∏—è
        –∑–¥–æ—Ä–æ–≤—å—è –Ω—É–∂–Ω–æ –Ω–µ —Ç–æ–ª—å–∫–æ –≤–ª–∞–¥–µ—Ç—å –Ω—É–∂–Ω–æ–π –∏–Ω—Ñ–æ—Ä–º–∞—Ü–∏–µ–π, –Ω–æ –∏ —É–º–µ—Ç—å —Ä–∞–∑—É–º–Ω–æ –µ–µ –ø—Ä–∏–º–µ–Ω–∏—Ç—å.
      </Typography>
      <Typography gutterBottom>
        –°–µ–π—á–∞—Å —è —É—á—É—Å—å –Ω–∞ health –∫–æ—É—á–∞. –°—É—Ç—å –º–æ–µ–π –ø—Ä–æ—Ñ–µ—Å—Å–∏–∏ - –ø–æ–º–æ—á—å —á–µ–ª–æ–≤–µ–∫—É –Ω–∞–π—Ç–∏ –±–∞–ª–∞–Ω—Å. –ë–∞–ª–∞–Ω—Å –≤ –µ–≥–æ —Ñ–∏–∑–∏—á–µ—Å–∫–æ–º,
        —ç–º–æ—Ü–∏–æ–Ω–∞–ª—å–Ω–æ–º, –º–µ–Ω—Ç–∞–ª—å–Ω–æ–º –∏ –¥—É—Ö–æ–≤–Ω–æ–º –∑–¥–æ—Ä–æ–≤—å–µ.
      </Typography>
      <Typography gutterBottom>
        –ú–æ—è —Ä–æ–ª—å –∫–∞–∫ health –∫–æ—É—á–∞ - –ø–æ–º–æ—á—å —á–µ–ª–æ–≤–µ–∫—É —É–≤–µ—Ä–µ–Ω–Ω–æ –ø—Ä–æ–π—Ç–∏ –µ–≥–æ –ø—É—Ç—å –∫ –æ–∑–¥–æ—Ä–æ–≤–ª–µ–Ω–∏—é, —Å–æ–∑–¥–∞–≤–∞—è –±–ª–∞–≥–æ–ø—Ä–∏—è—Ç–Ω—É—é
        –∞—Ç–º–æ—Å—Ñ–µ—Ä—É –¥–ª—è –∏–∑–º–µ–Ω–µ–Ω–∏–π, —Å–Ω–∞–±–∂–∞—è –Ω–µ–æ–±—Ö–æ–¥–∏–º—ã–º–∏ –∑–Ω–∞–Ω–∏—è–º–∏ –∏ –∏–Ω—Å—Ç—Ä—É–º–µ–Ω—Ç–∞–º–∏ –¥–ª—è –ø—Ä–∏–æ–±—Ä–µ—Ç–µ–Ω–∏—è, —Å–æ—Ö—Ä–∞–Ω–µ–Ω–∏—è –∏ —É–ø—Ä–∞–≤–ª–µ–Ω–∏—è
        —Å–≤–æ–∏–º –∑–¥–æ—Ä–æ–≤—å–µ–º.
      </Typography>
      <Typography gutterBottom>
        –ü–æ–∑–Ω–∞–∫–æ–º–∏—Ç—å—Å—è —Å–æ –º–Ω–æ–π –ø–æ–ª—É—á—à–µ –≤—ã –º–æ–∂–µ—Ç–µ —á–µ—Ä–µ–∑ –º–æ–π –ò–Ω—Å—Ç–∞–≥—Ä–∞–º{' '}
        <Link href="https://www.instagram.com/daria.klover">@daria.klover</Link> –∞–∫–∫–∞—É–Ω—Ç. –¢–∞–º —è –≤—Ä–µ–º—è –æ—Ç –≤—Ä–µ–º–µ–Ω–∏
        –≤—ã–∫–ª–∞–¥—ã–≤–∞—é —Å—Ç–æ—Ä–∏–∑ –∏ –ø–æ—Å—Ç—ã –æ —Å–µ–±–µ, —Å–≤–æ–∏—Ö –æ—Å–æ–∑–Ω–∞–Ω–∏—è—Ö –∏ –≤–∞–∂–Ω—ã—Ö –¥–ª—è –º–µ–Ω—è —Å–æ–±—ã—Ç–∏—è—Ö.
      </Typography>
      <Typography gutterBottom>–ê –¥–∞–Ω–Ω—ã–π —Å–∞–π—Ç - —ç—Ç–æ –ø–ª–æ—â–∞–¥–∫–∞ –¥–ª—è –ø—Ä–æ—Ñ–µ—Å—Å–∏–æ–Ω–∞–ª—å–Ω–æ–≥–æ —Ç–≤–æ—Ä—á–µ—Å—Ç–≤–∞.</Typography>
      <Typography gutterBottom>–î–æ–±—Ä–æ –ø–æ–∂–∞–ª–æ–≤–∞—Ç—å –≤ –≥–æ—Å—Ç–∏!</Typography>
    </>
  );
}

export default About;
