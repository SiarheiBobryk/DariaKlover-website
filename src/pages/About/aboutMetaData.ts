export interface AboutMetaData {
  title: string;
  heading: string;
  path: string;
}

const aboutMetaData: AboutMetaData = {
  title: 'Обо мне',
  heading: 'Обо мне',
  path: '/about',
};

export default aboutMetaData;
