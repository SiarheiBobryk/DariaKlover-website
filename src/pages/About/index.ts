export { default } from './About';
export { default as aboutMetaData } from './aboutMetaData';
