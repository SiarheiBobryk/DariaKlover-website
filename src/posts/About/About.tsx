import * as React from 'react';

import Typography from '@mui/material/Typography';
import Link from '@mui/material/Link';

import OpenInNewIcon from '@mui/icons-material/OpenInNew';
import { AppConfigContext, Config } from '../../providers/AppConfigProvider';
import Heading from '../../components/Heading';
import Paragraph from '../../components/Paragraph';

export const aboutConstants = {
  spell: `Ààkl…ô äv…ôr`,
};

export interface AboutProps {
  heading?: string;
}

function About(props: AboutProps) {
  const { heading } = props;
  const {
    instagram: { href: instagramHref, id: instagramId },
    fullName,
  }: Config = React.useContext(AppConfigContext);

  return (
    <>
      <Heading>{heading}</Heading>
      <Paragraph>–ü—Ä–∏–≤–µ—Ç! –Ø - {fullName}.</Paragraph>
      <Paragraph>
        –ö–ª–æ–≤–µ—Ä - –º–æ–π –ø—Å–µ–≤–¥–æ–Ω–∏–º. –û—Ç –∞–Ω–≥–ª–∏–π—Å–∫–æ–≥–æ <Typography component="i">{aboutConstants.spell}</Typography> - –∫–ª–µ–≤–µ—Ä üçÄ
      </Paragraph>
      <Paragraph>
        –ú–Ω–µ –æ—á–µ–Ω—å –Ω—Ä–∞–≤–∏—Ç—Å—è –∏–¥–µ—è –ø–µ—Ä–µ–¥–∞—á–∏ —Å–º—ã—Å–ª–æ–≤ —á–µ—Ä–µ–∑ –æ–±—Ä–∞–∑—ã. –í —ç—Ç–æ–º –µ—Å—Ç—å —á—Ç–æ-—Ç–æ —É–≤–ª–µ–∫–∞—Ç–µ–ª—å–Ω–æ–µ, –∏–Ω—Ç–µ—Ä–µ—Å–Ω–æ–µ –∏
        —Ç–∞–∏–Ω—Å—Ç–≤–µ–Ω–Ω–æ–µ. –£ –∫–ª–µ–≤–µ—Ä–∞ –µ—Å—Ç—å –º–Ω–æ–≥–æ –∑–Ω–∞—á–µ–Ω–∏–π. –î–ª—è –º–µ–Ω—è - —ç—Ç–æ –æ–ª–∏—Ü–µ—Ç–≤–æ—Ä–µ–Ω–∏–µ –±–∞–ª–∞–Ω—Å–∞ –∏ –≥–∞—Ä–º–æ–Ω–∏–∏. –ü–æ-–º–æ–µ–º—É, –∑–Ω–∞—á–µ–Ω–∏–µ
        —ç—Ç–æ–≥–æ —Å–∏–º–≤–æ–ª–∞ –æ—á–µ–Ω—å —Ö–æ—Ä–æ—à–æ –æ—Ç—Ä–∞–∂–∞–µ—Ç –º–µ–Ω—è –∏ —Ç–æ, –∫ –µ–º—É —è —Å—Ç—Ä–µ–º–ª—é—Å—å, –≤–æ —á—Ç–æ –≤–µ—Ä—é, —á–µ–º –∑–∞–Ω–∏–º–∞–ª–∞—Å—å –∏ –ø—Ä–æ–¥–æ–ª–∂–∞—é
        –∑–∞–Ω–∏–º–∞—Ç—å—Å—è.
      </Paragraph>
      <Paragraph>
        –í –ø—Ä–æ—à–ª–æ–º —è –≤—Ä–∞—á —Ç–µ—Ä–∞–ø–µ–≤—Ç, –≤—Ä–∞—á –æ–±—â–µ–π –ø—Ä–∞–∫—Ç–∏–∫–∏, –º–∞—Å—Å–∞–∂–∏—Å—Ç –∏ –∫–æ—Å–º–µ—Ç–∏–∫. –Ø –∑–Ω–∞—é –Ω–µ –ø–æ–Ω–∞—Å–ª—ã—à–∫–µ, —á—Ç–æ –¥–ª—è —Å–æ—Ö—Ä–∞–Ω–µ–Ω–∏—è
        –∑–¥–æ—Ä–æ–≤—å—è –Ω—É–∂–Ω–æ –Ω–µ —Ç–æ–ª—å–∫–æ –≤–ª–∞–¥–µ—Ç—å –Ω—É–∂–Ω–æ–π –∏–Ω—Ñ–æ—Ä–º–∞—Ü–∏–µ–π, –Ω–æ –∏ —É–º–µ—Ç—å —Ä–∞–∑—É–º–Ω–æ –µ–µ –ø—Ä–∏–º–µ–Ω–∏—Ç—å.
      </Paragraph>
      <Paragraph>
        –°–µ–π—á–∞—Å —è —É—á—É—Å—å –Ω–∞ health –∫–æ—É—á–∞. –°—É—Ç—å –º–æ–µ–π –ø—Ä–æ—Ñ–µ—Å—Å–∏–∏ - –ø–æ–º–æ—á—å —á–µ–ª–æ–≤–µ–∫—É –Ω–∞–π—Ç–∏ –±–∞–ª–∞–Ω—Å. –ë–∞–ª–∞–Ω—Å –≤ –µ–≥–æ —Ñ–∏–∑–∏—á–µ—Å–∫–æ–º,
        —ç–º–æ—Ü–∏–æ–Ω–∞–ª—å–Ω–æ–º, –º–µ–Ω—Ç–∞–ª—å–Ω–æ–º –∏ –¥—É—Ö–æ–≤–Ω–æ–º –∑–¥–æ—Ä–æ–≤—å–µ.
      </Paragraph>
      <Paragraph>
        –ú–æ—è —Ä–æ–ª—å –∫–∞–∫ health –∫–æ—É—á–∞ - –ø–æ–º–æ—á—å —á–µ–ª–æ–≤–µ–∫—É —É–≤–µ—Ä–µ–Ω–Ω–æ –ø—Ä–æ–π—Ç–∏ –µ–≥–æ –ø—É—Ç—å –∫ –æ–∑–¥–æ—Ä–æ–≤–ª–µ–Ω–∏—é, —Å–æ–∑–¥–∞–≤–∞—è –±–ª–∞–≥–æ–ø—Ä–∏—è—Ç–Ω—É—é
        –∞—Ç–º–æ—Å—Ñ–µ—Ä—É –¥–ª—è –∏–∑–º–µ–Ω–µ–Ω–∏–π, —Å–Ω–∞–±–∂–∞—è –Ω–µ–æ–±—Ö–æ–¥–∏–º—ã–º–∏ –∑–Ω–∞–Ω–∏—è–º–∏ –∏ –∏–Ω—Å—Ç—Ä—É–º–µ–Ω—Ç–∞–º–∏ –¥–ª—è –ø—Ä–∏–æ–±—Ä–µ—Ç–µ–Ω–∏—è, —Å–æ—Ö—Ä–∞–Ω–µ–Ω–∏—è –∏ —É–ø—Ä–∞–≤–ª–µ–Ω–∏—è
        —Å–≤–æ–∏–º –∑–¥–æ—Ä–æ–≤—å–µ–º.
      </Paragraph>
      <Paragraph>
        –ü–æ–∑–Ω–∞–∫–æ–º–∏—Ç—å—Å—è —Å–æ –º–Ω–æ–π –ø–æ–ª—É—á—à–µ –≤—ã –º–æ–∂–µ—Ç–µ —á–µ—Ä–µ–∑ –º–æ–π –ò–Ω—Å—Ç–∞–≥—Ä–∞–º{' '}
        <Link href={instagramHref} target="_blank">
          {instagramId} <OpenInNewIcon fontSize="small" sx={{ verticalAlign: 'middle' }} />
        </Link>
        . –¢–∞–º —è –≤—Ä–µ–º—è –æ—Ç –≤—Ä–µ–º–µ–Ω–∏ –≤—ã–∫–ª–∞–¥—ã–≤–∞—é —Å—Ç–æ—Ä–∏–∑ –∏ –ø–æ—Å—Ç—ã –æ —Å–µ–±–µ, —Å–≤–æ–∏—Ö –æ—Å–æ–∑–Ω–∞–Ω–∏—è—Ö –∏ –≤–∞–∂–Ω—ã—Ö –¥–ª—è –º–µ–Ω—è —Å–æ–±—ã—Ç–∏—è—Ö.
      </Paragraph>
      <Paragraph>–ê –¥–∞–Ω–Ω—ã–π —Å–∞–π—Ç - —ç—Ç–æ –ø–ª–æ—â–∞–¥–∫–∞ –¥–ª—è –ø—Ä–æ—Ñ–µ—Å—Å–∏–æ–Ω–∞–ª—å–Ω–æ–≥–æ —Ç–≤–æ—Ä—á–µ—Å—Ç–≤–∞.</Paragraph>
      <Paragraph>–î–æ–±—Ä–æ –ø–æ–∂–∞–ª–æ–≤–∞—Ç—å –≤ –≥–æ—Å—Ç–∏!</Paragraph>
    </>
  );
}

export default About;
