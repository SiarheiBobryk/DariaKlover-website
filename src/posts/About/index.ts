export { default } from './About';
export { default as aboutConstants } from './About';
