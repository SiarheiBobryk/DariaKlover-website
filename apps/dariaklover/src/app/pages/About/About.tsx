import * as React from 'react';
import { Helmet } from 'react-helmet';

import ImageList from '@mui/material/ImageList';
import ImageListItem from '@mui/material/ImageListItem';
import ImageListItemBar from '@mui/material/ImageListItemBar';
import Typography from '@mui/material/Typography';
import { Theme, useMediaQuery, useTheme } from '@mui/material';
import aboutMetaData from './aboutMetaData';

import aboutContent from './aboutContent.md';
import Markdown from '../../components/Markdown';

export const PUBLIC_PATH = 'assets/responsive_images';

// TODO: Review the image storage and remove unused images
const itemData = [
  {
    img: `${PUBLIC_PATH}/certificate-health-coach.jpg`,
    title: 'TODO: Add a title here',
    subtitle: 'TODO: Add a subtitle (if applicable)',
  },
  {
    img: `${PUBLIC_PATH}/diploma-health-coath.jpg`,
    title: 'TODO: Add a title here',
    subtitle: 'TODO: Add a subtitle (if applicable)',
  },
  {
    img: `${PUBLIC_PATH}/diploma-doctor-1.jpg`,
    title: 'TODO: Add a title here',
    subtitle: 'TODO: Add a subtitle (if applicable)',
  },
];

function About() {
  const theme: Theme = useTheme();
  const isMedium = useMediaQuery(theme.breakpoints.up('md'));
  const isSmall = useMediaQuery(theme.breakpoints.up('sm'));
  const isXSmall = useMediaQuery(theme.breakpoints.up('xs'));
  const columns = React.useMemo(
    function memoizeColumns() {
      if (isMedium) {
        return 3;
      }
      if (isSmall) {
        return 2;
      }
      if (isXSmall) {
        return 1;
      }
      return 3;
    },
    [isMedium, isSmall, isXSmall],
  );
  return (
    <>
      <Helmet>
        <title>{aboutMetaData.title}</title>
      </Helmet>
      <Markdown>{aboutContent}</Markdown>
      {/* TODO: Move it so a separate component */}
      <Typography component="h2" variant="h6">
        TODO: Add a heading here
      </Typography>
      {/* TODO: Remove default props */}
      <ImageList variant="standard" cols={columns} gap={8}>
        {itemData.map((item) => {
          return (
            // TODO: Remove default props
            // TODO: Make cols responsive
            <ImageListItem key={item.img} cols={1} rows={1}>
              {/* TODO: Add a skeleton maybe ðŸ¤” */}
              <img src={item.img} alt={item.title} loading="lazy" />
              <ImageListItemBar
                title={item.title}
                subtitle={item.subtitle}
                position="below"
                // actionPosition="right"
              />
            </ImageListItem>
          );
        })}
      </ImageList>
    </>
  );
}

export default About;
