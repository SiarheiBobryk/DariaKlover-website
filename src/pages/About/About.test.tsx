import * as React from 'react';
import { Helmet, HelmetPropsToState } from 'react-helmet';

import { axe, toHaveNoViolations } from 'jest-axe';
import { render, RenderResult } from '@testing-library/react';

import { typographyClasses } from '@mui/material/Typography';

import About from './About';
import aboutMetaData from './aboutMetaData';

expect.extend(toHaveNoViolations);

describe('<About />', () => {
  describe('a11y', () => {
    it('should match a11y standards', async () => {
      const { container }: RenderResult = render(<About />);

      const results = await axe(container);
      expect(results).toHaveNoViolations();
    });
  });

  describe('Elements inspection', () => {
    it('should have the right title HTML element', () => {
      render(<About />);

      const { title }: HelmetPropsToState = Helmet.peek();
      expect(title).toBe(aboutMetaData.title);
    });

    it('should have the `h2` HTML element as heading', () => {
      render(<About />);

      const h2: HTMLHeadingElement | null = document.querySelector('h2');
      expect(h2).toBeInTheDocument();
    });

    it('should have the CSS classes from `h5` Typography variant by default', () => {
      render(<About />);

      const h2: HTMLHeadingElement | null = document.querySelector('h2');
      expect(h2).toHaveClass(typographyClasses.h5);
    });

    it('should have the right constants', () => {
      render(<About />);

      const h2: HTMLHeadingElement | null = document.querySelector('h2');
      expect(h2).toHaveTextContent(aboutMetaData.heading);
    });
    // TODO: find out how to check the space char between the link here and for other component ðŸ¤”
  });
});
