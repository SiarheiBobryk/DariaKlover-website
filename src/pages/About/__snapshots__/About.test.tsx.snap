// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<About /> snapshotting should render correctly 1`] = `
Array [
  <h2
    className="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom css-h93ljk-MuiTypography-root"
  >
    –û–±–æ –º–Ω–µ
  </h2>,
  <p
    className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-d3wcwz-MuiTypography-root"
  >
    –ü—Ä–∏–≤–µ—Ç! –Ø 
    –î–∞—Ä—å—è –ö–ª–æ–≤–µ—Ä
    .
  </p>,
  <div
    className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation0 MuiAlert-root MuiAlert-standardInfo MuiAlert-standard css-1fuz9qr-MuiPaper-root-MuiAlert-root"
    role="alert"
  >
    <div
      className="MuiAlert-icon css-1ytlwq5-MuiAlert-icon"
    >
      <svg
        aria-hidden={true}
        className="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
        data-testid="InfoOutlinedIcon"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M11,9H13V7H11M12,20C7.59,20 4,16.41 4,12C4,7.59 7.59,4 12,4C16.41,4 20,7.59 20, 12C20,16.41 16.41,20 12,20M12,2A10,10 0 0,0 2,12A10,10 0 0,0 12,22A10,10 0 0,0 22,12A10, 10 0 0,0 12,2M11,17H13V11H11V17Z"
        />
      </svg>
    </div>
    <div
      className="MuiAlert-message css-1pxa9xg-MuiAlert-message"
    >
      <div
        className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom MuiAlertTitle-root css-yslrua-MuiTypography-root-MuiAlertTitle-root"
      >
        –ú–æ–µ —Ñ–æ—Ç–æ
      </div>
      –ü–æ–≤–µ—Ä—å—Ç–µ, —ç—Ç–æ —Å–ª–æ–∂–Ω–æ –ø–æ–¥–æ–±—Ä–∞—Ç—å –ø–æ–¥—Ö–æ–¥—è—â–µ–µ —Ñ–æ—Ç–æ –¥–ª—è —ç—Ç–æ–π —Å—Ç—Ä–∞–Ω–∏—Ü—ã, –Ω–æ –º—ã —Ä–∞–±–æ—Ç–∞–µ–º –Ω–∞–¥ —ç—Ç–∏–º –∏ —Å–∫–æ—Ä–æ –∏—Å–ø—Ä–∞–≤–∏–º—Å—è üôÇ
    </div>
  </div>,
  <p
    className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-d3wcwz-MuiTypography-root"
  >
    –ö–ª–æ–≤–µ—Ä - –∏–º—è –º–æ–µ–≥–æ –±—Ä–µ–Ω–¥–∞.
  </p>,
  <p
    className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-d3wcwz-MuiTypography-root"
  >
    –ü–æ—á–µ–º—É –ö–ª–æ–≤–µ—Ä? –ú–Ω–µ –Ω—Ä–∞–≤–∏—Ç—Å—è –ø–µ—Ä–µ–¥–∞–≤–∞—Ç—å —Å–º—ã—Å–ª—ã —á–µ—Ä–µ–∑ –æ–±—Ä–∞–∑—ã. –í —ç—Ç–æ–º –µ—Å—Ç—å —á—Ç–æ-—Ç–æ —É–≤–ª–µ–∫–∞—Ç–µ–ª—å–Ω–æ–µ, –∏–Ω—Ç–µ—Ä–µ—Å–Ω–æ–µ –∏ —Ç–∞–∏–Ω—Å—Ç–≤–µ–Ω–Ω–æ–µ. –£ –∫–ª–µ–≤–µ—Ä–∞ –µ—Å—Ç—å –º–Ω–æ–≥–æ –∑–Ω–∞—á–µ–Ω–∏–π. –î–ª—è –º–µ–Ω—è - —ç—Ç–æ –æ–ª–∏—Ü–µ—Ç–≤–æ—Ä–µ–Ω–∏–µ –±–∞–ª–∞–Ω—Å–∞ –∏ –≥–∞—Ä–º–æ–Ω–∏–∏. –ü–æ-–º–æ–µ–º—É, –∑–Ω–∞—á–µ–Ω–∏–µ —ç—Ç–æ–≥–æ —Å–∏–º–≤–æ–ª–∞ –æ—á–µ–Ω—å —Ö–æ—Ä–æ—à–æ –æ—Ç—Ä–∞–∂–∞–µ—Ç –º–µ–Ω—è –∏ —Ç–æ, –∫ –µ–º—É —è —Å—Ç—Ä–µ–º–ª—é—Å—å, –≤–æ —á—Ç–æ –≤–µ—Ä—é, —á–µ–º –∑–∞–Ω–∏–º–∞–ª–∞—Å—å –∏ –ø—Ä–æ–¥–æ–ª–∂–∞—é –∑–∞–Ω–∏–º–∞—Ç—å—Å—è.
  </p>,
  <p
    className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-d3wcwz-MuiTypography-root"
  >
    –í –ø—Ä–æ—à–ª–æ–º —è –≤—Ä–∞—á —Ç–µ—Ä–∞–ø–µ–≤—Ç, –≤—Ä–∞—á –æ–±—â–µ–π –ø—Ä–∞–∫—Ç–∏–∫–∏, –º–∞—Å—Å–∞–∂–∏—Å—Ç –∏ –∫–æ—Å–º–µ—Ç–∏–∫. –Ø –∑–Ω–∞—é –Ω–µ –ø–æ–Ω–∞—Å–ª—ã—à–∫–µ, —á—Ç–æ –¥–ª—è —Å–æ—Ö—Ä–∞–Ω–µ–Ω–∏—è –∑–¥–æ—Ä–æ–≤—å—è –Ω—É–∂–Ω–æ –Ω–µ —Ç–æ–ª—å–∫–æ –≤–ª–∞–¥–µ—Ç—å –Ω—É–∂–Ω–æ–π –∏–Ω—Ñ–æ—Ä–º–∞—Ü–∏–µ–π, –Ω–æ –∏ —É–º–µ—Ç—å —Ä–∞–∑—É–º–Ω–æ –µ–µ –ø—Ä–∏–º–µ–Ω–∏—Ç—å.
  </p>,
  <p
    className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-d3wcwz-MuiTypography-root"
  >
    –¢–µ–ø–µ—Ä—å —è –ø—Ä–∞–∫—Ç–∏–∫—É—é—â–∏–π health-–∫–æ—É—á - —Å–ø–µ—Ü–∏–∞–ª–∏—Å—Ç –ø–æ –∑–¥–æ—Ä–æ–≤–æ–º—É –æ–±—Ä–∞–∑—É –∂–∏–∑–Ω–∏ –∏ –Ω–µ–º–µ–¥–∏–∫–∞–º–µ–Ω—Ç–æ–∑–Ω–æ–º—É –æ–∑–¥–æ—Ä–æ–≤–ª–µ–Ω–∏—é.
  </p>,
  <p
    className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-d3wcwz-MuiTypography-root"
  >
    –ú–æ—è —Ä–æ–ª—å –∫–∞–∫ health –∫–æ—É—á–∞ - –ø–æ–º–æ—á—å —á–µ–ª–æ–≤–µ–∫—É —É–≤–µ—Ä–µ–Ω–Ω–æ –ø—Ä–æ–π—Ç–∏ –µ–≥–æ –ø—É—Ç—å –∫ –æ–∑–¥–æ—Ä–æ–≤–ª–µ–Ω–∏—é, —Å–æ–∑–¥–∞–≤–∞—è –±–ª–∞–≥–æ–ø—Ä–∏—è—Ç–Ω—É—é –∞—Ç–º–æ—Å—Ñ–µ—Ä—É –¥–ª—è –∏–∑–º–µ–Ω–µ–Ω–∏–π, —Å–Ω–∞–±–∂–∞—è –Ω–µ–æ–±—Ö–æ–¥–∏–º—ã–º–∏ –∑–Ω–∞–Ω–∏—è–º–∏ –∏ –∏–Ω—Å—Ç—Ä—É–º–µ–Ω—Ç–∞–º–∏ –¥–ª—è –ø—Ä–∏–æ–±—Ä–µ—Ç–µ–Ω–∏—è, —Å–æ—Ö—Ä–∞–Ω–µ–Ω–∏—è –∏ —É–ø—Ä–∞–≤–ª–µ–Ω–∏—è —Å–≤–æ–∏–º –∑–¥–æ—Ä–æ–≤—å–µ–º.
  </p>,
  <p
    className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-d3wcwz-MuiTypography-root"
  >
    –ü–æ–∑–Ω–∞–∫–æ–º–∏—Ç—å—Å—è —Å–æ –º–Ω–æ–π –ø–æ–ª—É—á—à–µ –≤—ã –º–æ–∂–µ—Ç–µ —á–µ—Ä–µ–∑ –º–æ–π –ò–Ω—Å—Ç–∞–≥—Ä–∞–º
     
    <a
      className="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-1ps4owl-MuiTypography-root-MuiLink-root"
      href="https://www.instagram.com/daria.klover"
      onBlur={[Function]}
      onFocus={[Function]}
      target="_blank"
    >
      @daria.klover
       
      <svg
        aria-hidden={true}
        className="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall css-1xzgpfl-MuiSvgIcon-root"
        data-testid="OpenInNewIcon"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M19 19H5V5h7V3H5c-1.11 0-2 .9-2 2v14c0 1.1.89 2 2 2h14c1.1 0 2-.9 2-2v-7h-2v7zM14 3v2h3.59l-9.83 9.83 1.41 1.41L19 6.41V10h2V3h-7z"
        />
      </svg>
    </a>
    . –¢–∞–º —è –≤—Ä–µ–º—è –æ—Ç –≤—Ä–µ–º–µ–Ω–∏ –≤—ã–∫–ª–∞–¥—ã–≤–∞—é —Å—Ç–æ—Ä–∏–∑ –∏ –ø–æ—Å—Ç—ã –æ —Å–µ–±–µ, —Å–≤–æ–∏—Ö –æ—Å–æ–∑–Ω–∞–Ω–∏—è—Ö –∏ –≤–∞–∂–Ω—ã—Ö –¥–ª—è –º–µ–Ω—è —Å–æ–±—ã—Ç–∏—è—Ö.
  </p>,
  <p
    className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-d3wcwz-MuiTypography-root"
  >
    –ê –¥–∞–Ω–Ω—ã–π —Å–∞–π—Ç - —ç—Ç–æ –ø–ª–æ—â–∞–¥–∫–∞ –¥–ª—è –ø—Ä–æ—Ñ–µ—Å—Å–∏–æ–Ω–∞–ª—å–Ω–æ–≥–æ —Ç–≤–æ—Ä—á–µ—Å—Ç–≤–∞.
  </p>,
  <p
    className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-d3wcwz-MuiTypography-root"
  >
    –î–æ–±—Ä–æ –ø–æ–∂–∞–ª–æ–≤–∞—Ç—å –≤ –≥–æ—Å—Ç–∏!
  </p>,
]
`;
